messages/training-checkout.txt

```diff
@@ -31,11 +45,3 @@ DC Lessons (from http://www.datacarpentry.org/lessons/): we would prefer contrib
 - OpenRefine-ecology: https://github.com/datacarpentry/OpenRefine-ecology/ 
 
 New Python lesson: https://github.com/swcarpentry/python-novice-gapminder.  This lesson is an experiment to see if we can start collaboration much earlier than we have in the past.  If you want to add to this, please submit an exercise that uses the Gapminder data set (in the "data" directory).  Please don't worry too much about formatting, and please only submit the Markdown files (not generated HTML) in pull requests.
-
-SWC Lessons (from http://software-carpentry.org/lessons/): these are pretty mature, and have had their tires kicked a lot by other trainees, so they need the least attention.  However, if you want to go through http://software-carpentry.org/blog/categories/#debriefing and pull out a few pieces of advice to add to the instructor's guide for any of these lessons, you would be my new best friend...
-
-- Unix shell: https://github.com/swcarpentry/shell-novice
-- Git: https://github.com/swcarpentry/git-novice
-- Python: https://github.com/swcarpentry/python-novice-inflammation
-- R: https://github.com/swcarpentry/r-novice-inflammation or https://github.com/swcarpentry/r-novice-gapminder
-- SQL: https://github.com/swcarpentry/sql-novice-survey
```

Comment:
I think we need somewhere the links to lessons which we want them to submit PRs to. In particular for SWC, the proliferation of repos is massive.